src/tests/Interop/PInvoke/IEnumerator/IEnumeratorNative.cpp

```diff
@@ -27,8 +27,8 @@ extern "C" DLL_EXPORT HRESULT STDMETHODCALLTYPE VerifyIntegerEnumerator(IEnumVAR
 
     VARIANT element;
     ULONG numFetched;
```

Comment:
```suggestion
    uint32_t numFetched;
```

Reply:
This needs to be `ULONG` since it is `unsigned long` on Windows.